src/console-services.yml

```diff
@@ -191,50 +415,98 @@
   description: 'S3 - Scalable Storage in the Cloud'
   url: 'https://console.aws.amazon.com/s3/home'
 
-- command: servermigration
-  description: 'Server Migration - Migrate on-premises servers to AWS'
-  url: 'https://console.aws.amazon.com/servermigration/home'
+- command: sagemaker
+  description: 'SageMaker - Build, Train, and Deploy Machine Learning Models'
+  url: 'https://console.aws.amazon.com/sagemaker/home'
+
+- command: secretsmanager
+  description: 'Secrets Manager - Easily rotate, manage, and retrieve secrets throughout their lifecycle'
+  url: 'https://console.aws.amazon.com/secretsmanager/home'
+
+- command: securityhub
+  description: 'Security Hub - AWS’s security and compliance center'
+  url: 'https://console.aws.amazon.com/securityhub/home'
+
+- command: serverlessrepo
+  description: 'Serverless Application Repository - Assemble, deploy, and share serverless applications within teams or publicly'
+  url: 'https://console.aws.amazon.com/serverlessrepo/home'
+
+- command: servermigrationservice
+  description: 'Server Migration Service - Migrate on-premises servers to AWS'
+  url: 'https://console.aws.amazon.com/servermigrationservice/home'
```

Comment:
this rename from `servermigration` --> `servermigrationservice` is not actually a valid link (it 404s for me)

Reply:
Fixed